### DISCUSSION TITLE
Example Folder Not Working Map Show Blank Brown

### DISCUSSION THREAD

**Discussion by putuoka** [react-native-maps/react-native-maps#4534] [Q&A]
Successfully Running example folder as the written tutorial but the map remain blank

![image](https://user-images.githubusercontent.com/20880839/203932377-75bebf42-99ef-4390-b1c9-5d6152317ce9.png)

**Comment by vinaciotm** [comment]
same here, any solution?

**Reply by dext36** [reply]
+1, changed API_KEY, Maps SDK for Android is enabled, unrestricted key, install by yarn, reinstalled with npm, still nothing..
Interested thing is I use same key for my referencial app based on react-native-maps and its working on any variation of environment.
The problem is somewhere in rn-maps repository. Its very easy to reproduce, please try to fix it..
